Java/core/src/main/java/com/amazon/randomcutforest/store/PointStoreDouble.java

```diff
@@ -48,8 +49,19 @@ public PointStoreDouble(boolean overlapping, int startOfFreeSegment, int dimensi
     }
 
     public PointStoreDouble(int dimensions, int capacity) {
-        super(dimensions, 1, capacity, false, true);
-        store = new double[capacity * dimensions];
+        this(new Builder().dimensions(dimensions).shingleSize(1).capacity(capacity).currentCapacity(capacity));
+    }
+
+    @Override
+    void resizeStore() {
+        int maxCapacity = rotationEnabled ? 2 * capacity : capacity;
```

Comment:
Does this mean that we don't support resizing when rotation is enabled. Would it be better to have a resize operation that fails when resizing is not allowed than to have one that silently does nothing?

Reply:
No, we do support resizing when rotation is enabled :)  Internally, we allow 2 times the (externally stated) capacity to allow for internally rotated shingling -- that is an artifact of this specific implementation, which allows better compression as a tradeoff. The issue is that a specific tuple cannot be copied to any location without losing the inherent intent in the position.